Submodels/file_handling.py

```
@@ -0,0 +1,685 @@
+                        # Compute the local sweep (i.e. LE offset) at the radial station from the provided interpolant
+                        sweep = blade_geometry["sweep_distribution"](radial_points[i]) 
+
+                        # Compute the thickness distribution at the radial station from the provided interpolant
+                        # Thickness is denormalised using the local chord length
+                        # Run check to ensure circumferential thickness does NOT exceed the limit of complete blockage.
+                        # If limit is exceeded, raises value error with radial point at which thickness was exceeded. 
+                        thickness_distribution = blade_geometry["thickness_distribution"]((radial_points[i], axial_points)) * local_chord   
+
+                        thickness_limit = 2 * np.pi * radial_points[i] / blading_params[stage]["blade_count"]
+                        if max(thickness_distribution) >= thickness_limit:
+                            raise ValueError(f"The cumulative blade thickness exceeds the complete blockage limit of 2PIr at r={radial_points[i]}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)